### ISSUE TITLE
Desktop app as flatpak

### ISSUE THREAD

**Issue by rugk** [rwv/lookscanned.io#209]
It would be nice if you could publish this as a [flatpak](https://flatpak.org/) on [flathub](https://flathub.org/) e.g.

Of course, this also requires the app to be wrapped in Electron or so, that you can easily install.
The advantage would be you can restrict it (by default) and not give it internet permission, so users can be sure, no data is sent anywhere. 

Flatpaks are a new software distribution mechanism for Linux distros, can thus installed on any distro and are easy to update. They are easy to install _and_ update and work on all Linux distros.
Also, if you publish it on _FlatHub_ you may grow your user base given many distros include that and it is a common software source, so your app can be discovered more easily. 

Here is [how to get started](http://docs.flatpak.org/en/latest/getting-started.html).

**Comment by rwv** [comment]
Electron is too heavy. Also PWA could provide app-like experience. What about your idea?

**Comment by rugk** [comment]
Yeah there are alternatives for Electron like https://tauri.app/ or so, but yeah, your choice. Personally, I've settled with having it as a PWS using GNOME Web (epiphany). That works and kinda has some desktop environment integration for me. Anyway, a release on flatpak could also bring popularity, as said, etc. So dunno…
